app/views/handlers/list_repeating_question.py

```diff
@@ -67,7 +67,10 @@ def get_previous_location_url(self) -> str:
     def handle_post(self) -> None:
         self.questionnaire_store_updater.add_completed_location(self.current_location)
```

Comment:
can't comment above, but as the current `self.repeating_block_ids` is incorrect, I think fix/remove it (probably remove)